doc/api/process.md

```diff
@@ -2228,6 +2228,13 @@ documentation for the [`'warning'` event][process_warning] and the
 [`emitWarning()` method][process_emit_warning] for more information about this
 flag's behavior.
 
+```console
+$ node --throw-deprecation -p "process.throwDeprecation"
+true
+$ node -p "process.throwDeprecation"
+undefined
+```
```

Comment:
It may be worthwhile to include an example that shows `process.throwDeprecation` is *mutable* at runtime...

```js
process.throwDeprecation = true;

process.emitWarning('test', 'DeprecationWarning');
// will cause an error to be thrown
```

Reply:
Totally worth it!

Reply:
Should docs changes? I mean, may say that `process. throwDeprecation` is mutable at runtime like: "The `process.throwDeprecation` is a mutable property at runtime which indicates..."

Reply:
That would be good, I think